modules/nf-core/svanalyzer/svbenchmark/environment.yml

```diff
@@ -0,0 +1,10 @@
+name: svanalyzer_svbenchmark
+
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+
+dependencies:
+  - bioconda::edlib=1.2.3
+  - bioconda::svanalyzer=0.35
```

Comment:
When making a module with more than one conda dependency, you must also create a mulled container with the same dependencies (to keep things reproducible), you can do this in this repository: https://github.com/BioContainers/multi-package-containers

Reply:
Or you can add it to the conda recipe, whatever you think is best

Reply:
Nice, it's not that dificult, you simply need to add it to the recipe (`meta.yaml`) found in this repository: https://github.com/bioconda/bioconda-recipes

Reply:
well I guess using version 0.36 was better idea since it already includes edlib.. https://github.com/bioconda/bioconda-recipes/blob/master/recipes/svanalyzer/meta.yaml 

So I decided to change it and keep it like that.